### PR TITLE
Update wavefront-opentracing-sdk-java to version 1.16

### PR DESCRIPTION
**Pull request by amitw-vmware**
Updating the version to 1.16.

### CODE REVIEW

File: src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java [hunk 1]
```diff
@@ -205,14 +206,11 @@ private void processResponse(ContainerRequestContext containerRequestContext,
...
       try {
-        Scope scope = (Scope) containerRequestContext.getProperty(PROPERTY_NAME);
-        if (scope != null) {
-          decorateResponse(containerResponseContext, scope.span());
-          scope.close();
-          scope.span().finish();
-        }
+        Span span = (Span) containerRequestContext.getProperty(PROPERTY_NAME);
+          decorateResponse(containerResponseContext, span);
+          span.finish();
```
Comment (hanwavefront) [inline src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java]: we should also close the scope, so i think we should pass along both scope and span in containerRequestContext

Comment (hanwavefront) [reply src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java]: also, nit: extra indent

**Review by hanwavefront — CHANGES_REQUESTED** [review]

File: src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java [hunk 2]
```diff
@@ -205,14 +206,11 @@ private void processResponse(ContainerRequestContext containerRequestContext,
...
     if (tracer != null) {
       try {
-        Scope scope = (Scope) containerRequestContext.getProperty(PROPERTY_NAME);
-        if (scope != null) {
-          decorateResponse(containerResponseContext, scope.span());
-          scope.close();
-          scope.span().finish();
-        }
+        Span span = (Span) containerRequestContext.getProperty(PROPERTY_NAME);
+          decorateResponse(containerResponseContext, span);
```
Comment (sushantdewan123) [inline src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java]: check if span is null

**Review by sushantdewan123 — CHANGES_REQUESTED** [review]

File: src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java (hunk 2 above)
Comment (amitw-vmware) [reply src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java]: fixed, thx!

File: src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java (hunk 1 above)
Comment (amitw-vmware) [reply src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java]: fixed, thx!

File: src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java [hunk 3]
```diff
@@ -205,14 +207,18 @@ private void processResponse(ContainerRequestContext containerRequestContext,
                                ContainerResponseContext containerResponseContext) {
     if (tracer != null) {
       try {
-        Scope scope = (Scope) containerRequestContext.getProperty(PROPERTY_NAME);
+        SpanWrapper spanWrapper = (SpanWrapper) containerRequestContext.getProperty(PROPERTY_NAME);
+        Scope scope = spanWrapper.getScope();
```
Comment (sushantdewan123) [inline src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java]: you need to check if spanWrapper is null

**Review by sushantdewan123 — APPROVED** [review]

File: src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java (hunk 3 above)
Comment (amitw-vmware) [reply src/main/java/com/wavefront/sdk/jaxrs/server/WavefrontJaxrsServerFilter.java]: fixed, thanks!

**Review by hanwavefront — APPROVED** [review]

**Review by sushantdewan123 — APPROVED** [review]

**Pull request merged**